LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Canonical Ltd.
+   Copyright 2023 Canonical Ltd.
```

Comment:
from the context of the text, my guess would be that it is not intended to change the license text at this point but keep it as a template:
> Copyright [yyyy] [name of copyright owner]

(from https://www.apache.org/licenses/LICENSE-2.0.txt)

Reply:
I think you are right, I will be revisiting this change. Thanks!

Reply:
Hey Michael, that text is intended to be changed: https://www.apache.org/licenses/LICENSE-2.0#apply

Reply:
No, it should not be changed. The LICENSE file itself should should be an unmodified copy of the Apache 2.0 license. The copyright and license file that is included should be applied to specific files within the source tree. From the text linked on how to apply, you will find the following (emphasis is mine):

> Include a *copy* of the Apache License, typically in a file called LICENSE, in your work, and consider also including a NOTICE file that references the License.

And then, it further goes on to state:

> To apply the Apache License to specific files in your work, attach the following boilerplate declaration, replacing the fields enclosed by brackets "[]" with your own identifying information. ...

As such, the LICENSE file should be an unaltered copy of the Apache 2.0 license and each file that will be licensed with the Apache License should include the boilerplate minimal license notice with the copyright information replaced.

cc: @NucciTheBoss

Reply:
Thanks for CCing - good thing I have a command-line app that prints out all the popular open-source licenses!